force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -11805,6 +11813,22 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>pmtModifyCurrency</shortDescription>
         <value>You can&apos;t modify Payment currencies directly. Update the parent Opportunity&apos;s currency, and NPSP will automatically update the currency of all related Payment records.</value>
     </labels>
+    <labels>
+        <fullName>pmtNoRefundPermissionMessage</fullName>
+        <categories>Payment Wizard</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Message displayed when the user has no permission to apply the Refund</shortDescription>
+        <value>You do not have permissions to Refund Payments.</value>
+    </labels>
+    <labels>
+        <fullName>pmtOriginalPaymentNotCompleted</fullName>
+        <categories>Payment</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Cannot refund the payment</shortDescription>
+        <value>The transaction is not completed.</value>
```

Comment:
@andrewyu-salesforce - When would this error get displayed? Is this related to permissions, or maybe an integration or server error where the transaction just couldn't complete for some reason?

Reply:
The error will get displayed when the user clicked 'refund' before Elevate processes the payment. It will get displayed when the refund API responds 422 Status Code.

Reply:
Ok, would this work?

"Payment can't be refunded because Elevate hasn't processed the payment yet."

Reply:
```suggestion
        <value>Payment can't be refunded because Elevate hasn't processed the payment yet.</value>
```